ci/get_labels/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/ops-agent/ci/get_labels"
+)
+
+type Test struct {
+	Name   string
+	Labels main.RespLabelCollection
+}
+
+func TestLabels(t *testing.T) {
+	testTable := []Test{
+		{
+			Name: "empty",
+		},
+		{
+			Name: "release & foo",
+			Labels: main.RespLabelCollection{
+				{
+					Name: "release",
+				},
+				{
+					Name: "foo",
+				},
+			},
+		},
+	}
+
+	for _, test := range testTable {
+		test := test
```

Comment:
This might not be necessary because this immediate `t.Run` isn't marked parallel.